tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -20,15 +20,26 @@ JupyterHub Spawner Is Visible
 Select Notebook Image
    [Documentation]  Selects a notebook image based on a partial match of ${notebook_image} argument
    [Arguments]  ${notebook_image}
-   Wait Until Element Is Visible  xpath:/html/body/div[1]/form/div/div/div[2]/div[2]/div[1]
+   ${version-check} =  Is RHODS Version Greater Or Equal Than  1.5.0
```

Comment:
since we are going to support the N-1 version, maybe it is worth to store the version in a common variable, so we don't need to change everywhere manually every release that requires a customization